go.mod

```diff
@@ -9,19 +9,17 @@ replace (
 
 require (
 	github.com/BurntSushi/toml v0.3.1 // indirect
+	github.com/Microsoft/go-winio v0.4.12
 	github.com/aws/aws-sdk-go v1.23.20
 	github.com/go-git/go-git/v5 v5.2.0
-	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/google/uuid v1.1.1
 	github.com/kardianos/service v1.0.0
-	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
```

Comment:
why are the indirect ones removed?  also, I guess go-windows-terminal-sequences is downgraded

Reply:
these were unused packages which got removed after running go mod tidy